webui_streamlit.py

```diff
@@ -1231,29 +1247,39 @@ def sample(init_data, x, conditioning, unconditional_conditioning, sampler_name)
 		output_images, info = None, None
 		history = []
 		initial_seed = None
+		
+		do_color_correction = False
+		try:
+			from skimage import exposure
+			do_color_correction = True
+		except:
+			print("Install scikit-image to perform color correction on loopback")		
 
-		for i in range(n_iter):
+		for i in range(1):
```

Comment:
This does not fix the loopback but simply circumvents it. Color correction code is thus pointless since the for loop only works once and the generated image is never fed to the sampler.

Essentially this is the same as `loopback = False`